### PR TITLE
gam 4.30 (new formula)

### PR DESCRIPTION
**Pull request by brianyoungblood**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/gam.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class Gam < Formula
+  desc "Command-line management for Google G Suite"
+  homepage "https://github.com/jay0lee/GAM/wiki"
+  url "https://github.com/jay0lee/GAM/archive/v4.30.tar.gz"
+  sha256 "846b2902fe36019a9f4ebd4a911e780e103fb2d210971b8a4e407ddb6d8db4f6"
+  head "https://github.com/jay0lee/GAM.git"
+
+  def install
+    prefix.install Dir["*"]
```
Comment (ilovezfs) [inline Formula/gam.rb]: This should use libexec

File: Formula/gam.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    bin.install_symlink prefix/"src/gam.py" => "gam"
+  end
+
+  test do
+    system "true"
```
Comment (ilovezfs) [inline Formula/gam.rb]: This needs to be a test that exercises and verifies the functionality of the software once it has been installed.

**Review by ilovezfs — CHANGES_REQUESTED** [review]
I'd like to see this use the upstream install script as well https://raw.githubusercontent.com/jay0lee/GAM/master/src/gam-install.sh.

**Comment by brianyoungblood** [comment]
@ilovezfs thanks for reviewing. I'll look into the code changes you requested for libexec and test. 

Regarding the request to see use of the upstream install script. The upstream script installs itself in your user's bin directory with an alias in the user's bash profile, which I didn't like.

```
# Update profile to add gam command
if [ "$update_profile" = true ]; then
  alias_line="alias gam=\"$target_dir/gam/gam\""
  if [ "$gamos" == "linux" ]; then
    update_profile "$HOME/.bashrc" || update_profile "$HOME/.bash_profile"
  elif [ "$gamos" == "macos" ]; then
    update_profile "$HOME/.profile" || update_profile "$HOME/.bash_profile"
  fi
else
  echo_yellow "skipping profile update."
fi
```

File: Formula/gam.rb (hunk 1 above)
Comment (brianyoungblood) [reply Formula/gam.rb]: @ilovefs this has been updated to use libexec

File: Formula/gam.rb (hunk 2 above)
Comment (brianyoungblood) [reply Formula/gam.rb]: @ilovezfs this has been updated with a basic test. although very basic.

**Comment by brianyoungblood** [comment]
@ilovezfs I've made the changes you requested for libexec and test. Thanks again for your review.

**Comment by brianyoungblood** [comment]
@ilovezfs is there anything else that requires work? Thank you.

**Comment by fxcoudert** [comment]
This will need to provide a real test, not merely outputting version number.

**Comment by brianyoungblood** [comment]
@fxcoudert I have now added a grep for the correct version number. Is there a formula that might be a better baseline to look at?

**Comment by fxcoudert** [comment]
@brianyoungblood the test should exercise the functionality of the tool. Perform an operation, and match the output or the resulting files.

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Pull request closed without merging**